### PR TITLE
Use links to aws bucket.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: _posts/2020-10-28-solidity-0.8.x-preview.md [hunk 1]
```diff
@@ -30,16 +30,16 @@ open an issue!
 
 You can find the preview binaries at the following locations:
 
- - [soljson.js](https://github.com/ethereum/solidity/releases/download/preview-0.8.x/soljson.js)
- - [solc-macos](https://github.com/ethereum/solidity/releases/download/preview-0.8.x/solc-macos)
- - [solc-static-linux](https://github.com/ethereum/solidity/releases/download/preview-0.8.x/solc-static-linux)
- - [solc.exe](https://github.com/ethereum/solidity/releases/download/preview-0.8.x/solc.exe)
+ - [soljson.js](https://solidity-blog.s3.eu-central-1.amazonaws.com/data/08preview/soljson.js )
```
Comment (cameel) [inline _posts/2020-10-28-solidity-0.8.x-preview.md:33]:
```suggestion
 - [soljson.js](https://solidity-blog.s3.eu-central-1.amazonaws.com/data/08preview/soljson.js)
```

**Review by franzihei — APPROVED** [review]

File: _posts/2020-10-28-solidity-0.8.x-preview.md [hunk 2]
```diff
@@ -30,16 +30,16 @@ open an issue!
...
+ - [solc-macos](https://solidity-blog.s3.eu-central-1.amazonaws.com/data/08preview/solc-macos)
+ - [solc-static-linux](https://solidity-blog.s3.eu-central-1.amazonaws.com/data/08preview/solc-static-linux)
+ - [solc.exe](https://solidity-blog.s3.eu-central-1.amazonaws.com/data/08preview/solc.exe)
 
 Note that all of these compilers will display a warning about them being a prerelease version, which is intentional.
 
 If you use solc-js, just copy the binary as ``soljson.js`` into the root directory of the solc-js npm module.
 
-You can also try out the 0.8.x compiler in [Remix](https://remix.ethereum.org/#version=https://github.com/ethereum/solidity/releases/download/preview-0.8.x/soljson.js). If you use this link, the preview compiler is already selected.
+You can also try out the 0.8.x compiler in [Remix](https://remix.ethereum.org/#version=https://solidity-blog.s3.eu-central-1.amazonaws.com/data/08preview/soljson.js). If you use this link, the preview compiler is already selected.
```
Comment (cameel) [inline _posts/2020-10-28-solidity-0.8.x-preview.md:42]: The compiler does not load when I use this link. I get this in the JS console in Firefox:
```
loading https://solidity-blog.s3.eu-central-1.amazonaws.com/data/08preview/soljson.js not allowed
```
I probably need to explicitly configure the CORS headers.

Comment (cameel) [reply _posts/2020-10-28-solidity-0.8.x-preview.md:42]: I configured CORS [just like for the `solc-bin` bucket](https://github.com/ethereum/solc-bin/pull/47#issuecomment-666447083) but it's still giving me this message...

Comment (cameel) [reply _posts/2020-10-28-solidity-0.8.x-preview.md:42]: The headers are clearly there so it must be something else:
```bash
curl --verbose --header "Origin: https://remix.ethereum.org" --header "Access-Control-Request-Method: GET" --header "Access-Control-Request-Headers: X-Requested-With" --request OPTIONS 'https://solidity-blog.s3.eu-central-1.amazonaws.com/data/08preview/soljson.js'```
```
```
> OPTIONS /data/08preview/soljson.js HTTP/1.1
> Host: solidity-blog.s3.eu-central-1.amazonaws.com
> User-Agent: curl/7.73.0
> Accept: */*
> Origin: https://remix.ethereum.org
> Access-Control-Request-Method: GET
> Access-Control-Request-Headers: X-Requested-With
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< x-amz-id-2: TRD3AeUQYejN4QBPyAvZ/xfqjsn+2xzUHiA8KrIlDUfJm/OPaORTKZXQPfbX4J1+B5AxDeT1xWM=
< x-amz-request-id: 43E855B8119EEDE5
< Date: Wed, 28 Oct 2020 12:54:25 GMT
< Access-Control-Allow-Origin: *
< Access-Control-Allow-Methods: GET, HEAD
< Access-Control-Allow-Headers: x-requested-with
< Vary: Origin, Access-Control-Request-Headers, Access-Control-Request-Method
< Content-Length: 0
< Server: AmazonS3
```

Comment (cameel) [reply _posts/2020-10-28-solidity-0.8.x-preview.md]: So, apparently you it's not possible to load a custom compiler like this.
See discussion: https://gitter.im/ethereum/solidity-dev?at=5f997cc22a35440715ee75bd

**Comment by cameel** [comment]
I just pushed a commit that adds instructions on how to load the custom compiler manually.

**Pull request merged**